debian/buster/foreman/foreman-plugin.mk

```diff
@@ -4,6 +4,7 @@ build:
 	cd /usr/share/foreman && ( \
 		/usr/bin/foreman-ruby /usr/bin/bundle install --local && \
 		if [ "$(PLUGIN_HAS_PACKAGE_JSON)" = "true" ]; then /usr/bin/npm install --no-audit --no-optional --unsafe-perm; fi && \
+		if [ "$(PLUGIN_NEEDS_GRAPHQL_WORKAROUND)" = "true" ]; then /usr/bin/npm install --save-dev 'graphql@^15.5.0' 'graphql-tag@^2.11.0'; fi && \
```

Comment:
I don't think we should codify this workaround so nicely, but rather fix the actual issue via https://github.com/theforeman/foreman/pull/8975